src/_guides/libraries/create-library-packages.md

```diff
@@ -205,7 +205,7 @@ or any `packages` directories. For more information, see
 
 {% comment %}
 PENDING: here just to make it easy to find discussions of `packages`...
-{% include coming-release.html %}
+{% include packages-dir.html %}
```

Comment:
The following files refer to packages-dir.html:

src/_guides/libraries/create-library-packages.md
src/_guides/libraries/private-files.md
src/_tutorials/libraries/shared-pkgs.md
src/tools/faq.md
src/tools/pub/cmd/pub-downgrade.md
src/tools/pub/cmd/pub-get.md
src/tools/pub/cmd/pub-upgrade.md
src/tools/pub/get-started.md
src/tools/pub/glossary.md
src/tools/pub/package-layout.md
src/tools/pub/publishing.md